lib/classes/Service.js

```diff
@@ -168,14 +169,22 @@ class Service {
     return this;
   }
 
-  setFunctionNames(rawOptions) {
+  ensureFunctionsAndSetNames(rawOptions) {
```

Comment:
I would opt-out from this rename. I believe it's not really necessary (?)

Note all this logic is scheduled to be trashed in close future, so I'd be surgical and apply only necessary changes.

Reply:
It's definitely not necessary :+1: I'll revert it